release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -438,6 +438,24 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-networking_{context}"]
 === Networking
 
+[id="ocp-ptp-fast-events-rest-api-v2-available_{context}"]
+==== New PTP fast events REST API v2 available
```

Comment:
Unless it's an official part of a product name, ISG prefers that "version" be spelled out at first use.